pkg/server/server.go

```diff
@@ -58,7 +58,7 @@ func init() {
 }
 
 func Run(version string, gitcommit string) {
-	log.Infof("Starting Spring Boot Generator Server on port - Version : %s (%s)",port,"/template/{id}",version,gitcommit)
+	log.Infof("Starting Spring Boot Generator Server on port %s, exposing endpoint %s - Version : %s (%s)",port,"/template/{id}",version,gitcommit)
```

Comment:
The endpoint /template/{id} is defined within the next line. So your proposition creates a redundancy ....

Reply:
OK, I'll update the PR.

Reply:
Fixed